tool/src/main/java/org/wildfly/security/tool/help/CommandsSection.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2022 Red Hat, Inc., and individual contributors
```

Comment:
Very minor, but we can update the year in the new files (sorry for the delayed review!).